rbi/annotations/paper_trail.rbi

```diff
@@ -0,0 +1,13 @@
+# typed: strict
+
+module PaperTrail::Model::ClassMethods
+  sig { params(options: T.nilable(T::Hash[Symbol, T.untyped])).void }
+  def has_paper_trail(options = {}); end
+
+  sig { returns(T.untyped) }
+  def paper_trail; end
```

Comment:
Can we get more precise here? It seems it returns a `PaperTrail::ModelConfig`: https://github.com/paper-trail-gem/paper_trail/blob/ccf0183088ed66c1604a0f825b560bdc7a125d88/lib/paper_trail/has_paper_trail.rb#L73